modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointment.rb

```diff
@@ -258,14 +253,7 @@ def status
         end
 
         def requested_periods
```

Comment:
I agree removing is changing functionality. We seem to look for requested periods in the reason_code whereas the service expects it to exist as an attribute. I'm not sure if that's a big deal or if they did it that way because requested periods no longer exist in the reason code

Reply:
Confirmed that the change this makes is just swapping the priority of reason code data vs attribute data. Should be good to go